### PR TITLE
Add executors FAQ

### PR DESCRIPTION
**Pull request by malomarrec**

### CODE REVIEW

File: content/departments/engineering/teams/batch-changes/go-to-market/server-side/index.md [hunk 1]
```diff
@@ -68,11 +68,36 @@ Yes. You can track this [here](https://sourcegraph.looker.com/explore/sourcegrap
...
+
+See [experimental and beta](https://docs.sourcegraph.com/admin/beta_and_experimental_features).
+
+#### Q: How do self-hosted customers deploy executors?
+
+See [deploying executors](https://docs.sourcegraph.com/admin/deploy_executors)
+
+#### How do Cloud customers use executors?
+
+Executors are provided by default to every cloud customer. What that means is thats erver-side batch changes is on by default for all cloud customers.
```
Comment (eseliger) [inline content/departments/engineering/teams/batch-changes/go-to-market/server-side/index.md]:
```suggestion
Executors are provided by default to every cloud customer. What that means is that server-side batch changes is on by default for all cloud customers.
```

**Comment by malomarrec** [comment]
@eseliger could you help me answering the unanswered question there? (and maybe adding other questions you got)

File: content/departments/engineering/teams/batch-changes/go-to-market/server-side/index.md [hunk 2]
```diff
@@ -68,11 +68,36 @@ Yes. You can track this [here](https://sourcegraph.looker.com/explore/sourcegrap
...
+#### Are there pricing implications for executors?
+
+- For self-hosted customers, executors are provided free of charge. Customers will have to self-host them though, which will cause infrastructure costs.
+- For cloud customers, 1 or more executors are included in their deployment by default, [depending on their ARR](https://github.com/sourcegraph/customer/issues/new?assignees=&labels=team%2Fcloud%2C+mi%2Cmi%2Fnew-instance-request&template=new_managed_instance.md&title=New+Managed+Instance+request%3A+%5BCUSTOMER+NAME%5D). If customers need more executors that included, this will come at an extra charge as defined in the Sourcegraph billing page.
+
+#### What are the most common complications/issues with deploying & using executors?
+
+#### How should issues with executors be escalated?
+
+Executors are beta, and we encounter very heterogenous customer setup, so it's exepcted that executor related questions will be escalated fast. Raise either via the normal channels, or in #wg-shipping-executors.
```
Comment (eseliger) [inline content/departments/engineering/teams/batch-changes/go-to-market/server-side/index.md]:
```suggestion
Executors are beta, and we encounter a very heterogenous customer setup, so it's expected that executor-related questions will be escalated fast. Raise either via the normal channels, or in #wg-shipping-executors.
```

File: content/departments/engineering/teams/batch-changes/go-to-market/server-side/index.md [hunk 3]
```diff
@@ -68,11 +68,39 @@ Yes. You can track this [here](https://sourcegraph.looker.com/explore/sourcegrap
...
+#### How do Cloud customers use executors?
+
+Executors are provided by default to every cloud customer. What that means is thats erver-side batch changes is on by default for all cloud customers.
+
+#### Are there pricing implications for executors?
+
+- For self-hosted customers, executors are provided free of charge. Customers will have to self-host them though, which will cause infrastructure costs.
+- For cloud customers, 1 or more executors are included in their deployment by default, [depending on their ARR](https://github.com/sourcegraph/customer/issues/new?assignees=&labels=team%2Fcloud%2C+mi%2Cmi%2Fnew-instance-request&template=new_managed_instance.md&title=New+Managed+Instance+request%3A+%5BCUSTOMER+NAME%5D). If customers need more executors that included, this will come at an extra charge as defined in the Sourcegraph billing page.
+
+#### What are the most common complications/issues with deploying & using executors?
```
Comment (eseliger) [inline content/departments/engineering/teams/batch-changes/go-to-market/server-side/index.md:102]:
```suggestion
#### What are the most common complications/issues with deploying & using executors?

Executors are best run on GCP or AWS using our provided terraform modules. If a customer can't or doesn't want to use terraform and one of these Cloud providers, they need to take manual steps, outlined in the docs. These may change over the releases, so this has to be monitored and adjusted for. In the future, we will provide a more straightforward non-cloud-specific setup.
When using firecracker KVM-level isolation (recommended for high security and the default on Sourcegraph Cloud), AWS instances need to be metal instances, as AWS doesn't support nested virtualization. On self-managed bare metal machines, KVM support needs to be available for this feature.
Customers have completely varying requirements for infrastructure, and it's likely we don't cover all of them yet (like custom encryption, networking tweaks, etc). The provided terraform modules are designed to be as flexible as possible, but we are also happy to add more customization options if they are generalizable. Some customers chose to fork our modules to extend them for their use cases as well.
```

**Review by eseliger — APPROVED** [review]
sound good?

**Pull request merged**